tsfile/src/main/java/org/apache/iotdb/tsfile/read/reader/chunk/ChunkReader.java

```diff
@@ -47,8 +47,7 @@
   private Filter filter;
 
   private BatchData data;
-
-  private long maxTombstoneTime;
+  protected long deletedAt;
```

Comment:
Make a specific comment about the deleteAt, maybe not in this class.
Variable `deletedAt` and `vision` is important. If the comment is good enough, we can just see these comments to know the implement details about delete function.

Reply:
Added a simple description. That is ideal, but impractical usually, otherwise we would not need documents.